app/api/csv/specs/csvLoader.spec.js

```diff
@@ -162,6 +169,22 @@ describe('csvLoader', () => {
       expect(textValues.length).toEqual(0);
     });
 
+    it('should arrange translations for selects and multiselects', async () => {
+      const trs = await translations.get();
+      trs.forEach(tr => {
+        expect(tr.contexts.find(c => c.label === 'thesauri1').values).toMatchObject({
+          thesauri1: 'thesauri1',
+          thesauri2: 'thesauri2',
+        });
+        expect(tr.contexts.find(c => c.label === 'multi_select_thesaurus').values).toMatchObject({
+          multi_select_thesaurus: 'multi_select_thesaurus',
+          multivalue1: 'multivalue1',
+          multivalue2: 'multivalue2',
+          multivalue3: 'multivalue3',
+        });
+      });
+    });
```

Comment:
im not sure why this is needed, i believe thesauri save already saves/updates the translation context ?

Reply:
Kept as it was a smoke test for an actual error that occurred. Moved out from this file to the other, thesauri related load tests.